src/gleam/dict.gleam

```diff
@@ -1,6 +1,6 @@
-import gleam/option.{type Option}
+import gleam/option.{type Option, None, Some}
 
-/// A dictionary of keys and values.
+/// A dictionary (dict) of keys and values.
```

Comment:
Should maybe dict in comments be replaced by `Dict`?

Reply:
Revert this please :pray: